docs/en/seeding-classic-image.md

```diff
@@ -0,0 +1,329 @@
+
+For background information on model assertions and signing them, see
+[image building][image-building]
+
+### Getting the brand SSO account assertion
+
+As noted, the brand SSO account is used to generate the key used to sign the
+model. Therefore, this account assertion needs to be seeded for the model to
+be authenticated.
+
+You can download this account assertion as follows:
+
+1. Login to snapcraft:
+
+        snapcraft login
+
+    And enter the Brand SSO account credentials.
+
+1. Display the SSO account’s account-id with `snapcraft whoami`:
+
+        snapcraft whoami
+        email:        THE-EMAIL
+        developer-id: THE-ACCOUNT-ID
+
+1. Download the brand SSO account assertion:
+
+        snap known account account-id=THE-ACCOUNT-ID > brand-account.assert
+
+### Getting the brand account-key assertion
+
+As noted, you need to obtain the account-key assertion for the key that signed
+the model. For this, you need the SHA3-384 fingerprint of the key. If the keys
+were generated on a system you can access, you can do the following on the
+system:
+
+1. Login to snapcraft using the brand SSO account details on the machine on
+which the model signing key was generated.
+1. Display known keys and copy the SHA3-384 fingerprint of the key used to
+sign the model with `snapcraft list-keys`:
+
+        snapcraft  list-keys
+            Name         SHA3-384 fingerprint
+        *   model     KEY-SHA3-384-FINGERPRINT
+        *   serial    KEY-SHA3-384-FINGERPRINT
+
+    If the keys were generated on another system, you need someone who can access that system provide the fingerprint.
+
+1. With the fingerprint, obtain the assertion as follows:
+
+        snap known account-key --remote public-key-sha3-384=KEY-SHA3-384-FINGERPRINT > brand-account-key.assert
+
+## Placing the seed directory into your image
+
+Now, simply copy the seed directory and all that it contains into your image
+in `/var/snapd/`, resulting in: `/var/snapd/seed/`.
+
+As noted, the seed directory and all files in it need to be owned by root
+user.  Recursively set ownership to root as follows:
+
+1. From the parent directory of seed/, change the owner recursively to root
+user as follows:
+
+        sudo chown -R root:root seed
+
+## Verifying seeding success
+
+You can verify success in a running image as follows:
+
+1. Verify the snaps you seeded are installed with:
+
+        snap list
+
+    You should see each snap you seeded as installed.
+
+1. Verify the model assertion with:
+
+        snap known model
+
+    You should see the model assertion you seeded.
+
+1. If your gadget snap includes a prepare-device hook in order to obtain a
+signed serial assertion from a serial vault, you can check the status with:
+
+        snap known serial
+
+    If there is no serial assertion, the transaction may be in progress, so use:
+
+        snap changes
+
+    Find the Initialize device change. If its status is not Done, note the ID and
```

Comment:
Mark "Initialize device" and "Done" as pre-formatted or bold or italics.

Reply:
I suppose they should be mono-space to indicate CLI output (or should it be bold as GUI quote? Nick?)

Reply:
we used <code> previously, I think it is appropriate here